tests/foreman/cli/test_satellitesync.py

```diff
@@ -230,11 +233,86 @@ def test_positive_export_rh_product(self):
         self.assertGreaterEqual(len(result.stdout), 1)
 
 
-class InterSatelliteSyncTestCase(CLITestCase):
-    """Implements InterSatellite Sync tests in CLI"""
+class ContentViewSync(CLITestCase):
+    """Implements Content View Export Import tests in CLI"""
+
+    @classmethod
+    def setUpClass(cls):
+        """Create Directory for all CV Sync Tests in /tmp"""
+        super(ContentViewSync, cls).setUpClass()
+        if ssh.command('[ -d /tmp/CvSync ]').return_code == 0:
+            ssh.command('rm -rf /tmp/CvSync')
+        ssh.command('mkdir /tmp/CvSync')
+        cls.exporting_org = make_org()
+        cls.exporting_prod = gen_string('alpha')
+        product = make_product({
+            'organization-id': cls.exporting_org['id'],
+            'name': cls.exporting_prod
+        })
+        cls.exporting_repo = gen_string('alpha')
+        repo = make_repository({
```

Comment:
Do we plan to implement other `ContentViewSync` test cases? if so, what is the reasoning behind putting this to `setUpClass`? what if we decide to implement the very same test as `test_positive_export_import_cv`, but with RH repo? isn't this something that should be handled by the individual tests?
I find this fixture kind of non-robust.

Reply:
> Do we plan to implement other `ContentViewSync` test cases? if so, what is the reasoning behind putting this to `setUpClass`?
There are such plans to add more tests. The reason behind putting things here is everytime I don't want to repeat the process of creating CV. I just want to have export and import in each test.
> what if we decide to implement the very same test as `test_positive_export_import_cv`, but with RH repo? isn't this something that should be handled by the individual tests?
I will mostly have tests around custom repo, and for RH repo I can create CV with RH repo for that perticular test only.
> I find this fixture kind of non-robust.
If I put setupClass contents for each test (just to avoid RH repo test), this will increase the time of execution.

Reply:
> Do we plan to implement other `ContentViewSync` test cases? if so, what is the reasoning behind putting this to `setUpClass`? what if we decide to implement the very same test as `test_positive_export_import_cv`, but with RH repo? isn't this something that should be handled by the individual tests?
There are such plans to add more tests. The reason behind putting things here is everytime I don't want to repeat the process of creating CV. I just want to have export and import in each test. I will mostly have tests around custom repo, and for RH repo I can create CV with RH repo for that perticular test only.
> I find this fixture kind of non-robust.
If I put setupClass contents for each test (just to avoid RH repo test), this will increase the time of execution.

Reply:
> Do we plan to implement other `ContentViewSync` test cases? if so, what is the reasoning behind putting this to `setUpClass`? what if we decide to implement the very same test as `test_positive_export_import_cv`, but with RH repo? isn't this something that should be handled by the individual tests?
> I find this fixture kind of non-robust.

There are such plans to add more tests. The reason behind putting things here is everytime I don't want to repeat the process of creating CV. I just want to have export and import in each test.
I will mostly have tests around custom repo, and for RH repo I can create CV with RH repo for that perticular test only.
If I put setupClass contents for each test (just to avoid RH repo test), this will increase the time of execution.

Reply:
thanks for clarification. I was trying to think something of by using pytest test case and the fixtures, but perhaps it wouldn't help much. let's keep it this way for the time being then.